bin/run-wer-automation.sh

```diff
@@ -32,7 +32,7 @@ export ds_validation_step
 ds_dropout_rate=0.05
 export ds_dropout_rate
 
-ds_default_stddev=0.1
+ds_default_stddev=0.046875
```

Comment:
I'd just remove `ds_default_stddev` from here, as it is super geometry related and shouldn't be tweaked from outside.